build.sh

```diff
@@ -38,10 +35,10 @@ function assert_path {
   fi
 }
 
-if [ -z "$JAVA_HOME" ]; then
-  echo "ERROR: JAVA_HOME not set! (maybe you need to do '. /etc/profile', or logout/login)"
-  exit 1
-fi
+# if [ -z "$JAVA_HOME" ]; then
```

Comment:
Hm. We do need a way to make sure that Java is present however. What platform are you building on?